Documentation/5.4.1/README.rst

```diff
@@ -160,9 +160,9 @@ The integration includes the following steps:
         - nuage-openvswitch (6wind version)
         - selinux-policy-nuage-avrs
         - python-pyelftools
-        - virtual-accelerator
-        - virtual-accelerator-addon-mellanox
         - virtual-accelerator-base
+        - virtual-accelerator (Only requried for VA version <= 1.8.3)
+        - virtual-accelerator-addon-mellanox (Only requried for VA version <= 1.8.3)
```

Comment:
virtual-accelerator-addon-mellanox isn't part of VA 1.8.3.
Might want to change it to :
(Only requried for VA version < 1.8.3)